src/new/wallet.js

```diff
@@ -2155,7 +2156,7 @@ class HathorWallet extends EventEmitter {
   getTokenData() {
     if (this.tokenUid === NATIVE_TOKEN_UID) {
       // Hathor token we don't get from the full node
-      this.token = HATHOR_TOKEN_CONFIG;
+      this.token = this.storage.getNativeTokenData();
```

Comment:
suggestion(non-blocking): It would be nice to have those two changed lines inside `src/new/wallet` tested as well, even though they're not as critical.

Reply:
Under codecov the facade lines seems to be tested